.tekton/build_and_trigger.yaml

```diff
@@ -85,56 +112,54 @@ spec:
         computeResources:
           requests:
             cpu: 1
+        runAfter:
+          - calculate-tag
+        taskRef:
+          name: buildah
+          kind: ClusterTask
         params:
           - name: IMAGE
             value: "quay.io/redhat-appstudio/clair-in-ci:$(tasks.calculate-tag.results.image_tag)"
           - name: BUILDER_IMAGE
             value: $(params.builder-image)
           - name: DOCKERFILE
             value: $(workspaces.source.path)/$(params.dockerfile)
-        runAfter:
-          - calculate-tag
-        taskRef:
-          name: buildah
-          kind: ClusterTask
-      - name: trigger-tests-and-push
-        computeResources:
-          requests:
-            cpu: 100mi
-        params:
-          - name: revision
-            value: "{{ revision }}"
-          - name: event_type
-            value: "{{ event_type }}"
-          - name: image_tag
-            value: $(tasks.calculate-tag.results.image_tag)
+      - name: test-clair
         runAfter:
           - build-image
         taskSpec:
-          params:
-            - name: revision
-              type: string
-            - name: event_type
```

Comment:
We need to keep this, since its being used in `login-and-push`.

Reply:
It should have this format imho:
```
          - name: event_type
            value: "{{ event_type }}"
```

Reply:
thanks